### PR TITLE
2.6 Migration docs fixups

### PR DESCRIPTION
**Pull request by gpgekko**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [ ] Have you [squashed your commits]? (Optional, but makes merge messages nicer.)
* [ ] Have you added copyright headers to new files?
* [ ] Have you checked that both Scala and Java APIs are updated?
* [ ] Have you updated the documentation for both Scala and Java sections?
* [ ] Have you added tests for any changed functionality?

# Helpful things

## Purpose

Fixing typos and various styling issues in the migration docs

## Background Context

Tried to fix everything I noticed while reading through the migration docs.

### CODE REVIEW

File: documentation/manual/releases/release26/migration26/JavaConfigMigration26.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ where `config` is your `com.typesafe.config.Config` instance.
 
 ## Manually checking values
 
-If you don't want or if you cannot have default values for some reason, you can use [`Config.hasPath`](https://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#hasPath-java.lang.String-) or [`Config.hasPathOrNull`](https://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#hasPathOrNull-java.lang.String-) to check if the value is configured before accessing it. This is a better option if the configuration is required but you can provide a reference (default) value:
+If you don't want or if you cannot have default values for some reason, you can use [`Config.hasPath`](https://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#hasPath-java.lang.String-) or [`Config.hasPathOrNull`](https://typesafehub.github.io/config/latest/api/com/typesafe/config/Config.html#hasPathOrNull-java.lang.String-) to check if the value is configured before accessing it. This is a better option if the configuration is required but you can't provide a reference (default) value:
```
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/JavaConfigMigration26.md]: I think this can should be a can't, so I changed it, but I have to say I find the whole sentence a bit vague and confusing so perhaps I'm misinterpreting and it wasn't a typo

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 2]
```diff
@@ -1199,8 +1159,7 @@ If you have custom cookies being used in Play, using the `CookieBaker[T]` trait,
 
 The `encode` and `decode` methods that `Map[String, String]` to and from the format found in the browser have been extracted into `CookieDataCodec`.  There are three implementations: `FallbackCookieDataCodec`, `JWTCookieDataCodec`, or `UrlEncodedCookieDataCodec`, which respectively represent URL-encoded with an HMAC, or a JWT, or a "read signed or JWT, write JWT" codec.
 
-
-and then provide a `JWTConfiguration` case class, using the `JWTConfigurationParser` with the path to your configuration, or use `JWTConfiguration()` for the defaults.
+You will also need to provide a `JWTConfiguration` case class, using the `JWTConfigurationParser` with the path to your configuration, or use `JWTConfiguration()` for the defaults.
```
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/Migration26.md]: It seems like someone removed a paragraph here, so I took the simple approach to turn it into a readable sentence, but this should probably be double checked by someone who has a clue as to what should be explained here...

**Comment by gpgekko** [comment]
I personally think the `Migration26.md` doc is to big for comfort. The order (or more the lack of order) of the numerous topics in it doesn't help either, making it hard to process / consume the info and scrolling to a section you've read before to check something a pain.
One thing that could be done is grouping the deprecation and removal sections a bit more.

File: documentation/manual/releases/release26/migration26/JavaConfigMigration26.md (hunk 1 above)
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/JavaConfigMigration26.md]: It should be a `can't`.

File: documentation/manual/releases/release26/migration26/MessagesMigration26.md [hunk 3]
```diff
@@ -306,7 +306,7 @@ class MessagesSpec extends PlaySpecification {
 
 ```
 
-If you need to customize the configuration, it's better to add configuration values into the GuiceApplicationBuilder rather than use the DefaultMessagesApiProvider directly.
+If you need to customize the configuration, it's better to add configuration values into the `GuiceApplicationBuilder` rather than use the `DefaultMessagesApiProvider` directly.
```
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/MessagesMigration26.md]: Could you add links to the APIs here too? Take a look at #7475 for reference.

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 4]
```diff
@@ -51,7 +51,7 @@ Play JSON has been moved to a separate library hosted at https://github.com/play
 libraryDependencies += "com.typesafe.play" %% "play-json" % "2.6.0"
 ```
 
-Also, play-json has a separate release cycle from the core Play library, so the version no longer is in sync with the Play version.
+Also, Play JSON has a separate release cycle from the core Play library, so the version is no longer in sync with the Play version.
```
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/Migration26.md]: We are using `play-json` almost everywhere, also, we are trying to use `play-ws`. So it is ok to keep it here as `play-json` too.

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 5]
```diff
@@ -129,10 +129,6 @@ Now, the default `Writeable[JsValue]` takes no implicit parameters and always wr
 
 If you need the old behavior back, you can define a `Writeable` with an arbitrary codec using `play.api.http.Writeable.writeableOf_JsValue(codec, contentType)` for your desired Codec and Content-Type.
 
-## Scala ActionBuilder and BodyParser changes
```
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Why did you remove this section?

File: documentation/manual/releases/release26/migration26/Migration26.md [hunk 6]
```diff
@@ -205,42 +201,6 @@ The Scala [`ActionBuilder`](api/scala/play/api/mvc/ActionBuilder.html) trait has
 
 The [`Action`](api/scala/play/api/mvc/Action$.html) global object and [`BodyParsers#parse`](api/scala/play/api/mvc/BodyParsers.html#parse:play.api.mvc.PlayBodyParsers) are now deprecated. They are replaced by injectable traits, [`DefaultActionBuilder`](api/scala/play/api/mvc/DefaultActionBuilder.html) and [`PlayBodyParsers`](api/scala/play/api/mvc/PlayBodyParsers.html) respectively. If you are inside a controller, they are automatically provided by the new [`BaseController`](api/scala/play/api/mvc/BaseController.html) trait (see [the controller changes](#Scala-Controller-changes) above).
 
-## Scala `Mode` changes
```
Comment (marcospereira) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Why did you remove this section?

File: documentation/manual/releases/release26/migration26/Migration26.md (hunk 5 above)
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/Migration26.md]: It is repeated [at line 202](https://github.com/playframework/playframework/pull/7486/commits/08f843cc7f402d6d92383ef326a71f895710b35c#diff-f1443d6313cdb479f5e6f01bbc48a780L202)

File: documentation/manual/releases/release26/migration26/Migration26.md (hunk 6 above)
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/Migration26.md]: It is repeated [at line 93](https://github.com/playframework/playframework/pull/7486/commits/08f843cc7f402d6d92383ef326a71f895710b35c#diff-f1443d6313cdb479f5e6f01bbc48a780L93)

File: documentation/manual/releases/release26/migration26/Migration26.md (hunk 4 above)
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/Migration26.md]: Ah, oke, I went with this because above the code example it seemed to be referenced to like that all the time. Will revert it.

File: documentation/manual/releases/release26/migration26/MessagesMigration26.md (hunk 3 above)
Comment (gpgekko) [inline documentation/manual/releases/release26/migration26/MessagesMigration26.md]: Done

**Comment by gpgekko** [comment]
Didn't see the email during the weekend, sadly. But I think I addressed all your points.

**Comment by marcospereira** [comment]
Thank you @gpgekko!

**Comment by marcospereira** [comment]
See backport here: #7503.

**Comment by Boomigha** [comment]
I have upgraded Play version from 2.5.10 to 2.6.6 in my application. While using the versions 2.6.X, inline CSS for HTML elements were not applied. But, it works on older versions 2.5.X. Anyone came across this issue ? Any suggestion on this?

Thanks in Advance :)

**Comment by marcospereira** [comment]
Hello @Boomigha,

Your problem may be related to some sbt-web plugin. Did you also update them?

Anyway, the [mailing list](https://groups.google.com/forum/#!forum/play-framework) would be a better place to discuss this. We use Github for reporting bugs and feature requests in Play, not for general discussion and getting help.
